ios/MobiusMobileWallet.xcodeproj/project.pbxproj

```diff
@@ -25,6 +24,20 @@
 		C2C418DC3CD144C6875373C8 /* Roboto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 89F9404328D54B39B885FE8B /* Roboto.ttf */; };
 		D042CDFFE93C4FFE9410B3C6 /* Helvetica Neue.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 97DB4B87A3CE4721975E2AB5 /* Helvetica Neue.ttf */; };
 		F6D0D10901774F07A8FBA2E3 /* Helvetica Neue Italic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D1A4434F98F742D9B1FA9409 /* Helvetica Neue Italic.ttf */; };
+		84ED744FA4CA412186B5DCC4 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D9BEF61E57D84FCEB3431366 /* Entypo.ttf */; };
```

Comment:
Seems like that changes not a part of your work, and I think it better to revert this file because of this unused fonts increases bundle size.

Reply:
Ive noticed this happens whenever I run yarn install. It also happened on Mateus' pr as well. Do you happen to know why?